app/Services/Trading212Service.php

```
@@ -0,0 +1,116 @@
+     */
+    public function fetchFilledOrdersSince(CarbonInterface $since): array
+    {
+        // If the official SDK is installed, use it. Otherwise, throw a helpful exception.
+        if (! class_exists('MarekSkopal\\Trading212\\Client')) {
+            Log::warning('Trading212 SDK not installed; please run composer install to fetch marekskopal/trading212.');
+
+            return [];
+        }
+
+        // Defer actual integration details to the SDK; keep normalization here.
+        $client = $this->makeSdkClient();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$client'. (undefined)

(UnusedLocalVariable)